static/scss/components/buttons.scss

```diff
@@ -41,3 +41,11 @@
     align-items: center;
     padding-left: $spacing-md;
 }
+
+.mzp-c-button.u-font-metropolis {
+    font-family: $font-metropolis;
+}
+
+.mzp-c-button.u-font-weight-normal {
+    font-weight: normal;
+}
```

Comment:
Since this is a utility class, I think you can move these over to [the `utils.scss` partial](https://github.com/mozilla/fx-private-relay/blob/rearrange-sign-up-btn/static/scss/includes/utils.scss) (and make it apply to other things outside of `.mzp-c-button`.

Reply:
FWIW – I was actually expecting more of a `.c-button-sign-in` instead of `.home-sign-in`. Good call on the utils class.